SETUP/check_require_login.php

```diff
@@ -115,6 +115,10 @@
     if(startswith($file, "vendor/"))
         continue;
 
+    # If it's in the node_modules directory, skip it
+    if(startswith($file, "node_modules/"))
```

Comment:
Apparently the flatted npm dependency required by has a php files in it:
https://github.com/WebReflection/flatted
Strange.
It is used by flat-cache which is required by eslint itself.

Reply:
Please go ahead and use the new if/then styling format even though it doesn't (yet) match the rest of the file:
```php
    // If it's in the node_modules directory, skip it
    if (startswith($file, "node_modules/")) {
        continue;
    }
```

That'll be 3 fewer lines that PHP-CS-Fixer needs to change 😁